oecologia-australis.csl

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-US" demote-non-dropping-particle="never">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
   <info>
     <title>Oecologia Australis</title>
     <title-short>OA</title-short>
     <id>http://www.zotero.org/styles/oecologia-australis</id>
     <link href="http://www.zotero.org/styles/oecologia-australis" rel="self"/>
     <link href="http://www.zotero.org/styles/apa" rel="template"/>
```

Comment:
why are you removing this? These seem like the most recent guidelines we have

Reply:
sorry, removing what? I didn´t understand your question.

Reply:
The link works: https://www.dropbox.com/s/azm8ycbkk229uce/Diretrizes%20OA%202017_12julho2017.pdf?dl=0

Reply:
Yes, but these are the old rules. I will include in some minutes one link to general journal guidelines. Thank you for calling my attention to this.